dist/testbed-support/makefile.iotlab.single.inc.mk

```diff
@@ -0,0 +1,190 @@
+# Use a single iot-lab node with RIOT
+# ===================================
+#
+# Supported:
+#  * flash
+#  * reset
+#  * term
+#
+# Tested on m3/a8-m3/wsn430/samr21/arduino-zero nodes
+#
+# It can be run:
+# * From your computer by setting IOTLAB_NODE to the full url like
+#   m3-380.grenoble.iot-lab.info or a8-1.grenoble.iot-lab.info
+# * From IoT-LAB frontend by setting IOTLAB_NODE to the short url like
+#   m3-380 or a8-1
+#
+# # Usage
+#
+#   make BOARD=iotlab-m3 IOTLAB_NODE=m3-380.grenoble.iot-lab.info flash term
+#
+# It is the user responsability to start an experiment beforehand
+#
+# If the user has multiple running experiments, the one to use must configured
+# with the `IOTLAB_EXP_ID` setting
+#
+# Prerequisites
+# -------------
+#
+# * Install iotlab-cli-tools: https://github.com/iot-lab/cli-tools
+# * Install iotlab-ssh-cli-tools: https://github.com/iot-lab/ssh-cli-tools
+#   * Can be done using given `requirements.txt`
+# * Configured your account with 'iotlab-auth'
+# * If using an A8 node from a frontend:
+#   * Register the frontend public key in your ssh keys list on your profile.
```

Comment:
Maybe give the tutorial link that explains this

Reply:
Which one is it ?

Also I expect people to know how to use IoT-LAB anyway for every other steps.

Note: it is only required because `iotlab-ssh` always uses an ssh gateway even when not required.

Reply:
When doing it from your computer it also requires that you add your personal key to the website…

Reply:
This one: https://www.iot-lab.info/tutorials/ssh-cli-client/

Reply:
I removed the mention on `requirements.txt` entirely.
My official documentation is use the IoT-LAB github documentation to install, I only added requirements.txt as version reference and for advanced users.